src/components/SelectionList/RadioListItem.tsx

```diff
@@ -40,13 +44,14 @@ function RadioListItem({
                 <View style={[styles.flex1, styles.alignItemsStart]}>
                     <TextWithTooltip
                         shouldShowTooltip={showTooltip}
-                        text={item.text}
+                        text={fullTitle}
                         textStyles={[
                             styles.optionDisplayName,
                             isFocused ? styles.sidebarLinkActiveText : styles.sidebarLinkText,
                             styles.sidebarLinkTextBold,
                             isMultilineSupported ? styles.preWrap : styles.pre,
                             item.alternateText ? styles.mb1 : null,
+                            isMultilineSupported ? {paddingLeft} : {},
```

Comment:
```suggestion
                            isMultilineSupported ? {paddingLeft} : null,
```

Not that it matters from the functionality standpoint, just to be consistent with the line above 

Other than that, looking good on all platforms!